src/test/unit/lang/stanc_helper_test.cpp

```diff
@@ -28,14 +28,14 @@ TEST(commandStancHelper, printStancHelp) {
 
 bool create_test_file(const std::string& path, const std::string& program) {
   std::string cmd_setup = "mkdir -p test/test-models";
-  system(cmd_setup.c_str());
+  int return_code = system(cmd_setup.c_str());
```

Comment:
This was added as g++ threw a warning that the return value is not used. And warnings are treated as errors.

Reply:
I think the usual solution if you want to ignore a return value is to cast it to void. I think that should also avoid the warning.